packages/flutter_tools/lib/src/runner/flutter_command_runner.dart

```diff
@@ -259,6 +264,12 @@ class FlutterCommandRunner extends CommandRunner<Null> {
       return;
     }
 
+    if (globalResults['version-json']) {
+      flutterUsage.sendCommand('version-json');
```

Comment:
Do we need to have separate analytics for this?  Seems like we should still report the same analytics and just move this into the `if (globalResults['version']) {...}` block